features/upgrade/node/hpa/upgrade.feature

```diff
@@ -0,0 +1,100 @@
+Feature: basic verification for upgrade testing
+  # @author weinliu@redhat.com
+  @upgrade-prepare
+  @users=upuser1
+  @admin
+  Scenario: Upgrade - Make sure hpa works well after upgrade - prepare
```

Comment:
Can you please update the scenario name for the test covers multiple resources? For example: `Upgrade - Make sure multiple resources works well after upgrade - prepare`

Reply:
Got you, updated

Reply:
The prepare step also needs updating